src/session-lock.c

```diff
@@ -152,12 +172,21 @@ static void
 align_session_lock_tree(struct output *output)
 {
 	struct wlr_box box;
-	wlr_output_layout_get_box(g_server->output_layout, output->wlr_output, &box);
+	wlr_output_layout_get_box(output->server->output_layout,
+		output->wlr_output, &box);
 	wlr_scene_node_set_position(&output->session_lock_tree->node, box.x, box.y);
 }
 
+static int
+handle_output_fill_timeout(void *data)
+{
+	struct session_lock_output *lock_output = data;
+	wlr_scene_node_set_enabled(&lock_output->background->node, true);
+	return 0;
+}
+
 void
-session_lock_output_create(struct session_lock *lock, struct output *output)
+session_lock_output_create(struct session_lock_manager *manager, struct output *output)
 {
 	struct session_lock_output *lock_output = znew(*lock_output);
 	if (!lock_output) {
```

Comment:
Is this check necessary? I don't think `lock_output` can be NULL here.

Reply:
Agree. `znew()` "handles" it.